client/src/routes.ts

```diff
@@ -1,31 +1,31 @@
-import Home from './features/home/Home';
-import AboutUs from './features/static/about/AboutUs';
-import HowItWorks from './features/static/about/HowItWorks';
-import OurStory from './features/static/about/OurStory';
-import TrustAndSafety from './features/static/policy/TrustAndSafety';
-import TermsOfService from './features/static/policy/TermsOfService';
-import PrivacyPolicy from './features/static/policy/PrivacyPolicy';
-import Login from './features/users/Auth/Login';
-import Signup from './features/users/Auth/Signup';
-import SignupCitizen from './features/users/Auth/SignupCitizen';
-import SignupNonProfit from './features/users/Auth/SignupNonProfit';
-import Inbox from './features/users/Inbox/Inbox';
-import User from './features/users/User';
-import ActionForm from './features/action/ActionForm';
-import Assets from './features/action/assets/Assets';
-import Asset from './features/action/assets/Asset';
-import NeedForm from './features/action/needs/NeedForm';
-import NeedFormGoods from './features/action/needs/NeedFormGoods';
-import NeedFormVolunteers from './features/action/needs/NeedFormVolunteers';
-import OfferForm from './features/action/offers/OfferForm';
-import OfferFormGoods from './features/action/offers/OfferFormGoods';
-import OfferFormSkills from './features/action/offers/OfferFormSkills';
-import ContactUs from './features/support/ContactUs';
-import Help from './features/support/Help';
-import ForgotPassword from './features/users/Auth/ForgotPassword';
-import SetNewPassword from './features/users/Auth/SetNewPassword';
-import CookiePolicy from './features/static/policy/CookiePolicy';
-import TempChat from './features/tempChat/TempChat';
+import Home from './views/Home';
+import AboutUs from './views/AboutUs';
+import HowItWorks from './views/HowItWorks';
+import OurStory from './views/OurStory';
+import TrustAndSafety from './views/TrustAndSafety';
+import TermsOfService from './views/TermsOfService';
+import PrivacyPolicy from './views/PrivacyPolicy';
+import Login from './components/Users/Auth/Login';
+import Signup from './components/Users/Auth/Signup';
+import SignupCitizen from './components/Users/Auth/SignupCitizen';
+import SignupNonProfit from './components/Users/Auth/SignupNonProfit';
+import Inbox from './components/Users/Inbox/Inbox';
+import User from './components/Users/User';
```

Comment:
the fact that they're in here tells me that these are views not components
some down below too